tests/jest/system/plugin-jar.spec.ts

```diff
@@ -188,11 +166,9 @@ async function assertFixture({
     options,
     mockSnykSearchClient,
   );
-  if (legacyPlugin.isMultiResult(result)) {
-    return t.fail('expected single inspect result');
-  }
-  const expected = depGraphLib.createFromJSON(
-    await readFixtureJSON(fixtureDirectory, 'dep-graph.json'),
-  );
-  t.ok(result.dependencyGraph?.equals(expected), 'dep-graphs are equal');
+  expect(legacyPlugin.isMultiResult(result)).toBeFalsy();
+  const expected = await readFixtureJSON(fixtureDirectory, 'dep-graph.json');
+  expect(
+    (result as legacyPlugin.SinglePackageResult).dependencyGraph!.toJSON(),
+  ).toEqual(expected);
```

Comment:
Here I’d leverage the DepGraph lib function that checks the equality of the graphs rather than toEqual